packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -290,13 +302,30 @@ export const Tooltip = styled.span< TooltipProps >`
 	) }
 `;
 
+const inputNumberWidth = ( { size }: InputNumberProps ) => {
+	const sizes = {
+		default: space( 16 ),
+		'__unstable-large': space( 20 ),
```

Comment:
Out of curiosity, is there a design spect about the number input's width changing too ?

Reply:
Maybe @jasmussen can give us a 👍  or a 👎  , since rich may be busy on other fronts

Reply:
I could be missing context, so I'll defer to Rich in case he chimes in.

But I don't recall anywhere we're using 20px for spacing items. We use 8px and 16px, in some places 12px, never 10 or 20.

Reply:
`space( 20 )` means `20 * 4px = 80px.

Prior to this PR, the input field was always `space( 16 )` (ie. `64px`) wide. Rich added a change that would make the input's width change to `80px` when its height is `40px`.

@jasmussen , we were wondering if it's ok for the input field to be wider when its height is `40px`, or if it should stay the same width as previously

Reply:
Ah, thanks for the clarification. In this case, anything goes, because I think for the particular opacity control in question, it's actually wrong as is, and should follow the pattern of the other more modern slides, a la radius:

<img width="379" alt="Screenshot 2023-06-07 at 16 02 03" src="https://github.com/WordPress/gutenberg/assets/1204802/2755ca3d-8031-43d1-b2ef-336f637dedc8">

But that's fine to do separately, there could be a valid discussion around that, so nothing to block work here.